core/parameters/res/runtime_configs/67.yaml

```diff
@@ -2,4 +2,4 @@ yield_resume: { old: false, new: true }
 wasm_yield_create_base: { old: 300_000_000_000_000, new: 153_411_779_276 }
 wasm_yield_create_byte: { old: 300_000_000_000_000, new: 15_643_988 }
 wasm_yield_resume_base: { old: 300_000_000_000_000, new: 1_195_627_285_210 }
-wasm_yield_resume_byte: { old: 300_000_000_000_000, new: 17_212_011 }
+wasm_yield_resume_byte: { old: 300_000_000_000_000, new: 1_195_627_285_210 }
```

Comment:
Why do you modify an old config? Even if unused it seems like a bad practice.

Reply:
IIUC we were processing this config file the wrong way because of the bug in [core/parameters/src/cost.rs](https://github.com/near/nearcore/pull/12192/files#diff-a89a413ba3059268467ad85cac0359ea435e6808a924712339c173f4d98d48b7). Now that we process it correctly we need to change the old file to maintain correct replayability.

Reply:
As described. The fee for byte between 67 and 73 was factually `1.195Tg` and this change reflects the reality. I could have added a new parameter altogether, but that would have required a change in 67 anyway (though it would have been entirely additive.)

Reply:
omg, of course, thanks for clarifying